cmd/boostd/init.go

```diff
@@ -382,6 +400,7 @@ func migrateMarketsConfig(cctx *cli.Context, mktsRepo lotus_repo.LockedRepo, boo
 		rcfg.LotusDealmaking = mktsCfg.Dealmaking
 		rcfg.LotusFees = mktsCfg.Fees
 		rcfg.DAGStore = mktsCfg.DAGStore
+		rcfg.DAGStore.RootDir = ""
```

Comment:
Given that some people will copy the dagstore over, whereas other people will move it, we should probably set this var depending on what they pick.